mithril-aggregator/src/file_uploaders/remote_file_uploader.rs

```diff
@@ -0,0 +1,86 @@
+use async_trait::async_trait;
+use slog::{debug, Logger};
+use std::path::Path;
+
+use mithril_common::logging::LoggerExtensions;
+use mithril_common::StdResult;
+
+use crate::file_uploaders::{FileLocation, FileUploader};
+
+/// RemoteSnapshotUploader is a snapshot uploader working using Google Cloud Platform services
+pub struct RemoteSnapshotUploader {
```

Comment:
You forgot to rename this `RemoteSnapshotUploader`

Reply:
`RemoteSnapshotUploader` was removed to use `GcpUploader` instead.